controller/Controller.php

```diff
@@ -517,4 +515,107 @@ function passwordPage($id)
         echo $view->render('views/password.html');
     }
 
+    /**
+     * Page to edit table items
+     *
+     * @param int $itemId The id of the item being edited
+     * @author Keller Flint
+     */
+    function editPage($itemId)
+    {
+        // Save Item
+        if (isset($_POST["itemSave"])) {
+            if ($this->_val->validateItem($_POST["itemName"])) {
+                $this->_db->updateItem($_POST["itemId"], $_POST["itemName"], $_POST["itemType"]);
+            } else {
+                $this->_f3->set("errors", $this->_val->getErrors());
+            }
+
+            if ($itemId == 0) {
```

Comment:
What's significant about itemID == 0? Just curious. Might be explained as I keep reviewing.

Reply:
Might have figured it out. Does it mean it's a new item?

Reply:
yes